packages/blockchain-wallet-v4-frontend/src/data/actions.js

```diff
@@ -24,6 +25,7 @@ export {
   goals,
   logs,
   form,
+  middleware,
```

Comment:
Middleware is a very specific concept in redux, that would be a confusing name
I'd suggest websockets or network or api

Reply:
We use websockets, api, and network in the core

Reply:
I see, at this level it makes sense I think indeed to rename that to webSocket, will change that!